.github/workflows/image.yaml

```diff
@@ -96,10 +96,11 @@ jobs:
         if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
         env:
           COSIGN_YES: true
-          IMAGE: "quay.io/kairos/kairos-${{ matrix.flavor }}:latest"
+          IMAGE: "quay.io/kairos/kairos-${{ matrix.flavor }}"
+          TAG: "latest"
```

Comment:
not sure I understand why we use latest here, while on the arm ones we use the GITHUB_REF

Reply:
changed to `GITHUB_REF` everywhere